packages/commonwealth/client/styles/components/component_kit/new_designs/CWSearchBar.scss

```diff
@@ -0,0 +1,155 @@
+@import '../../../shared';
+
+.Searchbar {
+  width: 608px;
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
+  padding: 8px 16px;
+  gap: 8px;
+  height: 40px;
+  background-color: $white;
+  border: 1px solid $neutral-200;
+  border-radius: 6px;
+  color: $neutral-800;
+
+  &.isTyping {
+    outline: none;
+    border-color: $primary-500;
+    box-shadow: 0px 2px 2px -1px rgba(0, 0, 0, 0.12),
+      0px 0px 0px 3px $primary-100;
+  }
+
+  &.disabled {
+    background-color: $neutral-50;
+    border-color: $neutral-200;
+    box-shadow: none;
+    color: $neutral-500;
+
+    &:hover {
+      border-color: $neutral-200;
+    }
+  }
+
+  &:hover {
+    border: 1px solid $primary-600;
+  }
+
+  input {
+    width: 100%;
+    border: none;
+
+    &:focus {
+      outline: none;
+    }
+
+    &.disabled {
+      background-color: $neutral-50;
+    }
+
+    ::placeholder {
+      color: $neutral-800;
+    }
+  }
+
+  &.magnifyingGlass {
+    color: $neutral-400;
+    width: fit-content;
+    padding: 0;
+    height: fit-content;
+    background-color: inherit;
+    border: none;
+    border-radius: 0;
+  }
+  
+  &.action {
+    display: grid;
+    align-items: center;
+    color: $neutral-500;
+    width: fit-content;
+    padding: 0;
+    height: fit-content;
+    background-color: inherit;
+    border: none;
+    border-radius: 0;
+  
+    &:hover {
+      cursor: pointer;
+      color: $neutral-600;
+    }
+  }
+  
+  &.inputElement {
+    flex: 1;
+    width: fit-content;
+    padding: 0;
+    height: fit-content;
+    background-color: inherit;
+    border: none;
+    border-radius: 0;
+  }
+}
+
+.container {
+  position: relative;
+  width: 608px;
+}
+
+.listBox {
+  position: absolute;
+  top: 100%;
+  left: 0;
+  width: 100%;
+  box-sizing: border-box;
+  padding: 6px;
+  margin-top: 0px;
+  max-height: 300px;
+  border-radius: 6px;
+  overflow: auto;
+  outline: 0px;
+  z-index: 1;
+  background-color: $white;
+  border: 1px solid #d0d7de;
+  color: #24292f;
+  box-shadow: 0px 4px 30px #d0d7de;
+}
+
+.option {
+  list-style: none;
+  padding: 8px;
+  border-radius: 8px;
+  cursor: default;
+
+  display: flex;
+  flex-direction: row;
+  gap: 8px;
+
+  font-family: 'NeueHaasUnica';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 20px;
+  align-items: center;
+  letter-spacing: 0.01em;
+  font-feature-settings: 'tnum' on, 'lnum' on;
+
+  &:hover {
+    cursor: pointer;
+    background-color: $neutral-100;
+  }
+}
+
+.noResults {
+  position: absolute;
+  top: 100%;
+  left: 0;
+  width: 100%;
+  margin-top: 0px;
+  padding: 14px;
+  max-height: 300px;
+  border-radius: 6px;
+  background-color: $white;
+  border: 1px solid #d0d7de;
+  color: #24292f;
+  box-shadow: 0px 4px 30px #d0d7de;
+}
```

Comment:
I think this is 6th time I put the same comment. All of these classes should be encapsulated because you are affecting other `container` and `option` classes

Reply:
Addressed.